polynote-frontend/polynote/comms.js

```diff
@@ -102,6 +105,10 @@ export class SocketSession extends EventTarget {
         }
     }
 
+    listenOnceFor(msgType, fn) {
```

Comment:
Forgot to actually use this, but it's to replace an anti-pattern where the listener removes itself (because it's trying to do a req-rep thing). Should also probably add a method explicitly for req-rep.

Reply:
Ah I see, so eventually the plan is to have more message listeners set `removeWhenFalse` to `true` rather than calling `removeMessageListener` directly?

Reply:
Yep, or they would use `listenOnceFor` instead.